docs/rules/no-large-snapshots.md

```diff
@@ -24,6 +24,9 @@ module.exports = {
 };
 ```
 
+In order to check external snapshots, you must also have `eslint` check files
+with the `.snap` extension by passing `--ext snap`.
```

Comment:
In addition, I eslint automatically lints files covered by `overrides` in config.

https://eslint.org/docs/latest/use/configure/configuration-files#specifying-target-files-to-lint

Reply:
This might have been what you meant, but even with `overrides` you still need to either provide `.snap` explicitly:

```
❯ tree -I node_modules
.
├── .eslintrc.js
├── __snapshots__
│   └── index.test.js.snap
├── index.test.js
├── package-lock.json
└── package.json
❯ cat .eslintrc.js
module.exports = {
  parserOptions: { ecmaVersion: 2020 },
  plugins: ['jest'],
  overrides: [
    {
      files: ['__snapshots__/*'],
      rules: {
        'jest/no-large-snapshots': ['error', { maxSize: 5 }]
      }
    }
  ]
};
❯ npx eslint .

❯ npx eslint . --ext .snap

/home/gareth/workspace/projects-scrap/snaptest/__snapshots__/index.test.js.snap
  3:1  error  Expected Jest snapshot to be smaller than 5 lines but was 7 lines long  jest/no-large-snapshots

✖ 1 problem (1 error, 0 warnings)
```

and likewise if I add `.snap` to the `'__snapshots__/*'` in the config, it works too.

I'll update the doc though to mention `overrides`